guides/common/modules/proc_creating-a-flatpak-remote.adoc

```diff
@@ -5,13 +5,19 @@
 
 You can create a Flatpak remote to access and manage Flatpak repositories in {Project}.
 
+.Prerequisites
+* Your {Project} account has the following permissions for Flatpak remotes: `create_flatpak_remotes`, `view_flatpak_remotes`, `edit_flatpak_remotes`, and `destroy_flatpak_remotes`.
+* Set up Flatpak on the content host that consumes applications from {ProjectServer}.
+For more information, see {RHELDocsBaseURL}9/html/administering_the_system_using_the_gnome_desktop_environment/assembly_installing-applications-using-flatpak_administering-the-system-using-the-gnome-desktop-environment#setting-up-flatpak_assembly_installing-applications-using-flatpak[Setting up Flatpak].
+* Ensure that Flatpak runtime repositories are available to clients alongside application repositories because Flatpak applications such as `rhel9/firefox-flatpak` depend on them for installation.
```

Comment:
```suggestion
* Ensure that Flatpak runtime repositories are available to clients alongside application repositories because Flatpak applications, such as `rhel9/firefox-flatpak`, depend on the runtime for installation.
```